internal/cloud/backend.go

```diff
@@ -600,16 +638,48 @@ func (b *Cloud) StateMgr(name string) (statemgr.Full, error) {
 	}
 
 	if err == tfe.ErrResourceNotFound {
```

Comment:
It looks like in the case when a resource it's found, meaning the workspace already exists, `terraform init` will output `Terraform Cloud has been successfully initialized!`. Which makes sense because the user should reinitialize as many times as they want (a pre-existing workspace). However, if the user were to run `terraform init` on a cloud block configuration that has a workspace name that exists and a project name that does not exist, then the output will also be `Terraform Cloud has been successfully initialized!`. I wonder if rather than silently fail at creating the new project, we should throw a warning. Thoughts?